CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Get upgrade notes from Sprockets 3.x to 4.x at https://github.com/rails/sprocket
 
 ## Master
 
+- Move to GitHub actions.[#709](https://github.com/rails/sprockets/pull/709)
```

Comment:
This CHANGELOG entry is not needed since this is not a change that the users should care about. Can you please remove?